ParadigApp/Assets/Scripts/ChangeScene.cs

```diff
@@ -7,20 +7,24 @@ public class ChangeScene : MonoBehaviour {
 
 	static List<string> scenes = new List<string> ();
 
-	public void changeToScene(string pScene){
+	public void changeToScene(string pScene)
+	{
```

Comment:
What is the `p` as prefix of names? if that means "parameter", remove it, please. Here and in all occurrences.